src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Machine {
+    public String doWork() {
+        return "The machine starts working...";
+    }
+
+    public String stopWork() {
+        return "The machine has finished working";
+    }
+}
```

Comment:
methods are not abstract
![image](https://github.com/mate-academy/jv-oop/assets/73041091/924253f4-64f8-4361-9042-6044481cb76d)